### DISCUSSION TITLE
Moving world files

### DISCUSSION THREAD

**Discussion by westpfelia** [Nimdy/Dedicated_Valheim_Server_Script#42] [Q&A]
Hey I was also wondering if I wanted to import world files from another server into this script what would be the best way to do that? 

Should I just rename all the new world files to what the script created and delete the old ones? or can I edit the script to start using a different set of files?

**Comment by westpfelia** [comment]
Ok, so following this issue request: https://github.com/Nimdy/Dedicated_Valheim_Server_Script/issues?q=is%3Aissue+is%3Aclosed

I was able to just rename the files in my worlds folder. and when I connect to the server the world itself looks right and my character spawns in at the right spot. But none of the buildings we made have loaded in and the day is set to 1. Character skills and inventory is still correct. 

Any idea?

**Reply by Nimdy** [reply]
yis...... check the valheim_start.sh file
your worldname names do not match
or 
your permissions are not set to steam:steam on the files you transferred over.

**Comment by ansund** [comment]
I'm followint the "Migrate Valheim Map Data from server to server" tutorial, 
when I'm trying to upload old worldfiles to  sterver using WINscp I get following error:
`Permission denied.
Error code: 3
Error message from server: Permission denied`

What should I do?

**Reply by Nimdy** [reply]
Login as root?

**Reply by ansund** [reply]
ah, no not ligedin as root in  WinSCP.
Solved it by chaning owner of worlds folder to my user, moving files, and then changing back owner to steam. 
Probably not the best solution but it worked for moving the files.

**Reply by ansund** [reply]
The rest worked like a charm. MANY THANKS! You rock.

**Reply by Nimdy** [reply]
no problem!
